src/plugins/auto/tests/unit/runtime_fallback_test.cpp

```diff
@@ -328,7 +331,10 @@ TEST_P(AutoCTPUTRuntimeFallback, ctputDeviceInferFailTest) {
                                                                                              mockExecutor,
                                                                                              nullptr,
                                                                                              ifThrow);
-            ON_CALL(*mockIExeNet.get(), create_infer_request()).WillByDefault(Return(mockInferrequest));
+            ON_CALL(*mockIExeNet.get(), create_infer_request()).WillByDefault([this]() {
+                std::this_thread::sleep_for(std::chrono::milliseconds(0));
```

Comment:
is this sleep needed?

Reply:
I'm not sure. I add it because I copy the template of other lambda in context. Honestly, I don't really know what effect it will have in this lambda.

Reply:
I think we can remove this sleep line